tests/ignite/metrics/test_confusion_matrix.py

```diff
@@ -7,8 +7,8 @@
 
 from ignite.exceptions import NotComputableError
 from ignite.metrics import ConfusionMatrix, IoU, mIoU
-from ignite.metrics.confusion_matrix import cmAccuracy, cmPrecision, cmRecall
-
+from ignite.metrics.confusion_matrix import cmAccuracy, cmPrecision, cmRecall, dice_coefficient
+import random
```

Comment:
Maybe we can use `torch.rand` instead of `random`?

Reply:
Hi @vfdev-5 , since we are using numpy array, I have used random here.

Reply:
In this case you can directly `np.random`